package/test/index.test.ts

```diff
@@ -170,6 +171,47 @@ describe('requestHandler', () => {
     });
   });
 
+  /**
+   * Test excessive pagination returns 400
+   */
+
+  it('should return 400', async () => {
```

Comment:
Consider a better `it should` description; eg, insert the text from the above comment here.

Reply:
Good shout, fixed on https://github.com/dekimir/postgres-searchbox/pull/12